lib/host/canvas.ex

```diff
@@ -0,0 +1,101 @@
+defmodule Inky.Host.Canvas do
```

Comment:
Adding the host subdirectory feels like an unnecessarily hierarchical choice. I suggest we keep things flat until there are many of the "it"s we want to group.

Reply:
I would rather silo the host code which would include HostCommands or whatever the module might be called.  Mostly because it is irrelevant to understanding most of the library, it is off the beaten path, it is an extra thing. Honestly it might be relevant to make it a separate lib to avoid it getting compiled to nerves builds and stuff.
So it has a reason. I want people to be able to grasp the important code of the project at a glance and ignore this.
Does that make sense as a use for a subdirectory for you?

Reply:
Hm... those are good points about the file layout saying guiding the reader's eyes, but I think your idea of making it a separate lib would be even better, then it can be a dev-dependency, just as you said!

Side-note: `HostCommands` sounds like a good name since it's the commands specific to the host-target :)

Reply:
Yeah, I figure since we have TestIO, RpiIO and if I had done the IO bit for this it would have been HostIO. So I imagine something similar will be done for the commands?

Reply:
Pretty much, yeah :)